### ISSUE TITLE
java/Ice/ami test hanging (Windows, CI)

### ISSUE THREAD

**Issue by bernardnormier** [zeroc-ice/ice#3044]
From https://github.com/zeroc-ice/ice/actions/runs/11646353744/job/32430345213:

```
process C:\hostedtoolcache\windows\Java_Oracle_jdk\17\x64\bin\java -ea -Dtest.class=test.Ice.ami.Client test.TestDriver --Ice.Default.Host=::1 --Test.BasePort=14100 --Ice.Warn.Connections=1 --Ice.Default.Protocol=ws --Ice.IPv6=1 --Ice.PreferIPv6Address=1 --Ice.Admin.Endpoints="tcp -h \"::1\"" --Ice.Admin.InstanceName=Client --IceMX.Metrics.Debug.GroupBy=id --IceMX.Metrics.Parent.GroupBy=parent --IceMX.Metrics.All.GroupBy=none --Ice.Trace.Network=2 --Ice.Trace.Retry=1 --Ice.Trace.Protocol=1 --Ice.LogFile=D:\a\ice\ice\java\test\src\main\java\test\Ice\ami\client-110224-2338.log pid=7472 is hanging - 11/02/24 23:39:55
```

many times and eventually times out. Unfortunately we don't gather log for tests that are hanging.

**Comment by bernardnormier** [comment]
From the message, it's an ipv6 + ws + mx flavor of this test,  whereas #3049 is ipv4 + wss.

**Comment by pepone** [comment]
> Unfortunately we don't gather log for tests that are hanging.

We do, but it somehow didn't work here, the upload logs step was run but it didn't detect any log files.

**Comment by pepone** [comment]
Most likely fixed by #3142, let's reopen if it still happens with the new code.
